include/flexflow/graph.h

```diff
@@ -200,6 +227,20 @@ class SearchHelper {
   template <typename T>
   void add_operator_cost(NodeAssignment const &, float, T *) const;
 
+  template <typename T>
+  void add_sink_node_costs(NodeAssignment const &sink,
+                           CostMetrics metrics,
+                           T *result) const;
+
+  /**
+   * @brief Add run time cost and memory cost of the operator to the graph cost.
```

Comment:
Can you add a brief explanation of why this is only temporary and what needs to be done to refactor it into a finished version?

Reply:
Oh, I just meant that we should revisit the functions once the complete version of search algorithm is done. It's just a reminder to revisit the whole search procedure once the code is stable.

Reply:
Okay, some context for that would be great. Can you create an issue for tracking that and add links? Otherwise it'll be too easy to forget what this meant and why it's there